### PR TITLE
ADM-359.1 Refactor

### PR DESCRIPTION
**Pull request by LanxinWang**
## Summary

refactor code

### CODE REVIEW

File: frontend/__tests__/src/components/metrics/ConfigStep/Board.test.tsx [hunk 1]
```diff
@@ -177,18 +177,6 @@ describe('Board', () => {
     })
   })
 
-  it('should check loading animation when click verify button', async () => {
```
Comment (guzhongren) [inline frontend/__tests__/src/components/metrics/ConfigStep/Board.test.tsx:180]: we should leave this test to cover `close loading state`

**Review by guzhongren — COMMENTED** [review]
Need change

**Review by spadio9587 — COMMENTED** [review]
little suggestion for spelling

**Review by tingwang1234 — APPROVED** [review]
LGTM

**Review by TingLiMio — APPROVED** [review]
LGTM

**Pull request merged**